package.json

```diff
@@ -1,32 +1,38 @@
 {
-  "name": "laminas-ci-matrix-action",
-  "engines": {
-    "node": "^18"
-  },
-  "type": "commonjs",
-  "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "^5.16.0",
-    "@typescript-eslint/parser": "^5.16.0",
-    "@types/node": "^18.6.4",
-    "eslint": "^8.11.0",
-    "eslint-config-incredible": "^2.4.2",
-    "eslint-import-resolver-typescript": "^2.7.0",
-    "eslint-plugin-import": "^2.25.4",
-    "eslint-plugin-node": "^11.1.0",
-    "source-map-loader": "^3.0.1",
-    "typescript": "~4.6.3",
-    "webpack": "^5.70.0",
-    "webpack-cli": "^4.9.2"
-  },
-  "scripts": {
-    "lint": "eslint src/ --ext .ts",
-    "lint-fix": "eslint src/ --ext .ts --fix",
-    "build": "tsc --build && webpack --mode=production"
-  },
-  "dependencies": {
-    "@actions/core": "^1.6.0",
-    "@cfworker/json-schema": "^1.12.2",
-    "@types/semver": "^7.3.9",
-    "semver": "^7.3.5"
-  }
+    "name": "laminas-ci-matrix-action",
+    "description": "Create a CI matrix for use in a GitHub Action",
+    "scripts": {
+        "build": "tsc --build && webpack --mode=production",
+        "lint": "eslint src/ --ext .ts",
+        "lint-fix": "eslint src/ --ext .ts --fix",
+        "test": "jest --logHeapUsage"
+    },
+    "engines": {
+        "node": "^18"
+    },
+    "dependencies": {
+        "@actions/core": "^1.10.0",
+        "@cfworker/json-schema": "^1.12.5",
+        "@types/semver": "^7.3.13",
+        "semver": "^7.3.8"
+    },
+    "devDependencies": {
+        "@types/jest": "^29.2.0",
+        "@types/node": "^18.11.7",
+        "@types/webpack": "^5.28.0",
+        "@typescript-eslint/eslint-plugin": "^5.41.0",
+        "@typescript-eslint/parser": "^5.41.0",
+        "eslint": "^8.26.0",
+        "eslint-config-incredible": "^2.4.2",
+        "eslint-import-resolver-typescript": "^3.5.2",
+        "eslint-plugin-import": "^2.26.0",
+        "eslint-plugin-node": "^11.1.0",
+        "jest": "^29.2.2",
+        "source-map-loader": "^4.0.1",
+        "ts-jest": "^29.0.3",
+        "ts-node": "^10.9.1",
+        "typescript": "~4.8.4",
+        "webpack": "^5.74.0",
```

Comment:
Just realized: we don't really webpack anything, do we? 🤔

Reply:
The `build` command in scripts runs `tsc --build && webpack --mode=production`, which is called from `Dockerfile`.